edx_proctoring/views.py

```diff
@@ -199,7 +200,7 @@ def get(self, request, course_id, content_id):
         attempt_data = {}
         active_exam = {}
 
-        is_learning_mfe = True if request.GET.get('is_learning_mfe') in ['1', 'true', 'True', True] else False
+        is_learning_mfe = request.GET.get('is_learning_mfe') in ['1', 'true', 'True', True]
```

Comment:
`True` object cannot be received from the request.GET dict like object it is always a string:
```suggestion
        is_learning_mfe = request.GET.get('is_learning_mfe') in ['1', 'true', 'True']
```